form/events.rst

```diff
@@ -175,13 +178,14 @@ transforms back the normalized data to the model and view data.
 
 It can be used to change data from the normalized representation of the data.
 
-===============  ===================================================================================
-Data Type        Value
-===============  ===================================================================================
-Model data       Same as in ``FormEvents::POST_SET_DATA``
-Normalized data  Data from the request reverse-transformed from the request using a view transformer
```

Comment:
This is reported by the issue. Indeed this value is not yet in `$event->getForm()->getNormData()` but it's given in `$event->getData()`